src/scss/components/header/_my-lists.scss

```diff
@@ -0,0 +1,147 @@
+@import "./../../base/variables";
+@import "./../../base/mixins";
+
+.my-lists {
+  $root: &;
+  $listWidth: 250px;
+  $borderWidth: 1px;
+  $index: 1;
+  $throttle: .25s;
+
+  // NOTE: I believe this would be inside flex container
+  // So no need to use float
+  float: right;
+
+  &__container {
+    display: none;
+  }
+
+  &__icon,
+  &__indicator {
+    vertical-align: middle;
+  }
+
+  &__icon {
+    width: 35px;
+    height: 35px;
+    fill: $gray;
+  }
+
+  &__indicator {
+    display: none;
+  }
+
+  @include desktop {
+    position: relative;
+    @include font-size(14px);
+
+    &__direct {
+      display: inline-block;
+      position: relative;
+      z-index: $index + 1;
+      border: $borderWidth solid transparent;
+      border-radius: $globalBorderRadius $globalBorderRadius 0 0;
+      padding: 6px;
+      text-decoration: none;
+      transition: border-color, $throttle, linear;
```

Comment:
this only delay showing dropdown without special effects and it's not related to showing cart dropdown. Looks like slower then cart dropdown.